src/directives/internal/class.js

```diff
@@ -38,7 +38,17 @@ export default {
   handleArray (value) {
     this.cleanup(value)
     for (var i = 0, l = value.length; i < l; i++) {
-      if (value[i]) {
+      if (value[i] && isPlainObject(value[i])) {
+        var keys = Object.keys(value[i])
+        for (var j = 0; j < keys.length; j++) {
```

Comment:
this part has a lot of duplicate with `handleObject`. we should extract them into a common function.